data/gqlHelper.js.njk

```diff
@@ -40,14 +40,14 @@ const helperSingleton = {
     {% endfor %}
   }`,
 
-  explorerPageQuery: graphql`query gqlHelperExplorerPageQuery{
+  datasetBrowserPageQuery: graphql`query gqlHelperDatasetBrowserPageQuery{
```

Comment:
why is this file changing? isn't this file auto generated

Reply:
Hmm I don't think it's autogenerated -- I think it is the template for autogeneration of gqlHelper.js. The comment at the top of gqlHelper.js says that this file generated it.